xbmc/pvr/PVRGUIActions.h

```diff
@@ -233,6 +233,15 @@ namespace PVR
     bool PlayRecording(const CFileItemPtr &item, bool bCheckResume) const;
 
     /*!
+     * @brief Play epg tag.
+     * @param item containing a an epg tag.
+     * @param bPlayMinimized controls whether the recording should be played fullscreen or in a window.
+     * @param bCheckResume controls resume check.
+     * @return true on success, false otherwise.
+     */
+    bool PlayEpgTag(const CFileItemPtr &item, bool bPlayMinimized, bool bCheckResume) const;
```

Comment:
No need for playminimized param. If we do this right, this will be handled internally by CPVRGUIActions.